client.go

```diff
@@ -183,4 +248,4 @@ func (c Client) RunWithInput(command string, stdout, stderr io.Writer, stdin io.
 	cmd.Wait()
 
 	return cmd.ExitCode(), cmd.err
-}
+}
```

Comment:
I know this isn't really an issue anymore, but please leave the last `\n` in place (this is valid for the other files below), as at least it silences git and/or github diff.